make-release.sh

```diff
@@ -71,18 +71,6 @@ else
     # add git tag
     git tag "${NEW_VERSION}"
 
-    # change back manifests to :latest
-    find "${REPO_ROOT}/kubernetes-manifests" -name '*.yaml' -exec sed -i -e "s/:${NEW_VERSION}/:latest/g" {} \;
-    find "${REPO_ROOT}/loadgenerator-manifests" -name '*.yaml' -exec sed -i -e "s/:${NEW_VERSION}/:latest/g" {} \;
-    git add "${REPO_ROOT}/kubernetes-manifests/*.yaml"
-    git add "${REPO_ROOT}/loadgenerator-manifests/*.yaml"
-
-    # change back telemetry Pub/Sub topic to "Test" topic
-    sed -i -e "s/topic_id = \"${PROD_TOPIC}\"/topic_id = \"${TEST_TOPIC}\"/g" ${REPO_ROOT}/terraform/telemetry.py;
-    git add "${REPO_ROOT}/terraform/telemetry.py"
```

Comment:
How is this going to be handled in the "new world? Do we have a configuration that distinguishes test/prod environments?

Reply:
In develop, we use `test` and on main, we use `prod`. I'm going to make another PR with a new GitHub Action to enforce this